backend/Models/Data/Sighting.cs

```diff
@@ -0,0 +1,21 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+
+namespace WhaleSpotting.Models.Data;
+
+public class Sighting
+{ 
+    [Key]
+    public int Id { get; set; }
+    public int UserId { get; set; }
+    public int WhaleSpeciesId { get; set; }
+    public float Lattitude { get; set; }
+    public float Longitude { get; set; }
+    public int AreaNameId { get; set; }
+    public string PhotoUrl { get; set; }
+    public string Description { get; set; }
```

Comment:
It looks like the description should be optional from the image on the ticket, did you make a conscious decision to change it or is this just a typo?
I can see either way being sensible I just want to make sure it's intentional